ReactCommon/yoga/yoga/Yoga.h

```diff
@@ -45,12 +45,12 @@ typedef struct YGValue {
 extern const YGValue YGValueUndefined;
 extern const YGValue YGValueAuto;
 
-#ifdef __cplusplus
+// #ifdef __cplusplus
 
-extern bool operator==(const YGValue& lhs, const YGValue& rhs);
-extern bool operator!=(const YGValue& lhs, const YGValue& rhs);
+// extern bool operator==(const YGValue& lhs, const YGValue& rhs);
+// extern bool operator!=(const YGValue& lhs, const YGValue& rhs);
```

Comment:
@MaxRis - I have done this to avoid a Qt error stating something like duplicate `operator` definition.  What is the proper way to do this?

Reply:
Here's the exact error: `.\ReactCommon\yoga\yoga\Yoga.h:50: error: conflicting declaration of C function 'bool operator==(const YGValue&, const YGValue&)'`.

This code also exists in Facebook sources: https://github.com/facebook/react-native/blob/2151c30474518858a5368013f5ea024588b279b9/ReactCommon/yoga/yoga/Yoga.h#L48-L53.